--- a/feeds/daed/daed/Makefile
+++ b/feeds/daed/daed/Makefile
@@ -10,6 +10,6 @@ CORE_VERSION:=core-caa6f5e
 WING_HASH_SHORT:=$(shell echo $(WING_VERSION) | cut -d- -f2)
 CORE_HASH_SHORT:=$(shell echo $(CORE_VERSION) | cut -d- -f2)
-PKG_RELEASE:=1
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_PROTO:=git
@@ -62,7 +62,7 @@ define Package/daed
   TITLE:=A Modern Dashboard For dae
   DEPENDS:=$(GO_ARCH_DEPENDS) \
     +ca-bundle +kmod-sched-core +kmod-sched-bpf \
-    +kmod-veth +v2ray-geoip +v2ray-geosite \
+    +kmod-veth +daed-geodata-bundle \
     +@KERNEL_XDP_SOCKETS \
     +DAED_USE_VMLINUX_BTF:vmlinux-btf
 endef
@@ -102,11 +102,15 @@ define Build/Prepare
 		rm -rf $(DAED_BUILD_DIR) ; \
 		mkdir -p $(DAED_BUILD_DIR) ; \
 		$(TAR) --strip-components=1 -C $(DAED_BUILD_DIR) -xzf $(DL_DIR)/$(PKG_NAME)-$(PKG_VERSION).tar.gz ; \
+		patch -d $(DAED_BUILD_DIR) -p1 < \
+			$(CURDIR)/files/h5000m-web-defaults.patch || exit 1 ; \
 		rm -rf $(PKG_BUILD_DIR)/dae-core ; \
 		git clone https://github.com/daeuniverse/dae $(PKG_BUILD_DIR)/dae-core && \
 			git -C $(PKG_BUILD_DIR)/dae-core checkout $(CORE_HASH_SHORT) ; \
+		patch -d $(DAED_BUILD_DIR) -p1 < \
+			$(CURDIR)/files/h5000m-runtime-optimizations.patch || exit 1 ; \
 		pushd $(PKG_BUILD_DIR)/dae-core ; \
 			git submodule update --init ; \
 		popd ; \
