--- a/feeds/daed/daed/files/daed.init
+++ b/feeds/daed/daed/files/daed.init
@@ -37,6 +37,7 @@ start_service() {
 	procd_append_param command --logfile-maxbackups "$log_maxbackups"
 	procd_append_param command --logfile-maxsize "$log_maxsize"
 
+	procd_set_param env DAE_LOCATION_ASSET="/usr/share/v2ray"
 	procd_set_param limits core="unlimited"
 	procd_set_param limits nofile="1000000 1000000"
 	procd_set_param respawn
--- a/feeds/daed/luci-app-daed/root/etc/init.d/luci_daed
+++ b/feeds/daed/luci-app-daed/root/etc/init.d/luci_daed
@@ -64,7 +64,6 @@ restore_resolv_conf() {
 }
 
 start_service() {
-  [ -f "/etc/init.d/daed" ] && grep -q "DAE_LOCATION_ASSET" "/etc/init.d/daed" || sed -i '/run/i\  procd_set_param env DAE_LOCATION_ASSET="/usr/share/v2ray"' "/etc/init.d/daed"
   config_load "$CONF"
 
   local enabled
